src/rust/engine/fs/store/src/lib.rs

```diff
@@ -1203,7 +1188,6 @@ impl Store {
     destination: PathBuf,
     digest: DirectoryDigest,
     mutable_paths: &BTreeSet<RelativePath>,
-    immutable_inputs: Option<&ImmutableInputs>,
```

Comment:
Aren't immutable inputs still necessary for large input directories (i.e., those which don't contain any large files, but which contain many small files)?

Reply:
We punted on that because it was causing issues. If a future author wants to add it they can :smile: